soh/soh/Extractor/Extract.cpp

```diff
@@ -492,13 +500,13 @@ extern "C" int zapd_main(int argc, char** argv);
 
 bool Extractor::CallZapd() {
     constexpr int argc = 16;
-    char xmlPath[100];
-    char confPath[100];
+    char xmlPath[1024];
+    char confPath[1024];
```

Comment:
Not opposed to this change, just curious if there's any particular reason for it. And frankly, is there any reason these aren't `std::strings`?

Reply:
I didn't write the original code so I would not know the reason behind not using `std::strings`. But as for the increase in size I was having paths that were longer than 100 characters.

Reply:
I thought that was probably the case. Seemed like an odd restriction to begin with. `std::string` would remove any hard size limitations like the above but we can wait to worry about that if we end up having issues with paths greater than 1024 characters.